data/sql/updates/pending_db_world/rev_1589121823177850900.sql

```diff
@@ -0,0 +1,224 @@
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 9454;
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 9454);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(9454, 0, 0, 0, 0, 0, 100, 0, 2200, 2400, 15400, 15600, 11, 13443, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Xavathras - In Combat - Cast \'13443\''),
+(9454, 0, 1, 0, 2, 0, 100, 1, 5, 30, 0, 0, 11, 8599, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Xavathras - Between 5-30% Health - Cast \'8599\' (No Repeat)');
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 7110;
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 7110);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(7110, 0, 0, 0, 11, 0, 100, 0, 0, 0, 0, 0, 11, 11013, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Jadefire Shadowstalker - On Respawn - Cast \'11013\''),
+(7110, 0, 1, 0, 4, 0, 100, 0, 0, 0, 0, 0, 11, 13578, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Jadefire Shadowstalker - On Aggro - Cast \'13578\''),
+(7110, 0, 2, 0, 0, 0, 100, 0, 2600, 3100, 19800, 20100, 11, 14897, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Jadefire Shadowstalker - In Combat - Cast \'14897\'');
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 7106;
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 7106);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(7106, 0, 0, 0, 4, 0, 100, 0, 0, 0, 0, 0, 11, 13578, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Jadefire Rogue - On Aggro - Cast \'13578\''),
+(7106, 0, 1, 0, 2, 0, 100, 1, 15, 30, 0, 0, 0, 13579, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Jadefire Rogue - Between 15-30% Health - No Action Type (No Repeat)');
```

Comment:
1. This makes no sense and is causing errors
2. A complete version already exists in the DB.

What I suggest:
Check the other SAI's if they already exist in DB and remove this one

Reply:
Sorted.

Sadly, we can't use Gauge cause it make problems.